[Map Changes] Reparaciones Delta y Meta

Spircen opened the pull request:
## What Does This PR Do
Aquí agregare todos los reportes que reciba de Delta y Meta en un periodo de 1 semana.
Delta:

- Repara cable faltante a APC de Comms
- Repara acceso de maint a seguridad
- Reagrega el Autolathe a recibidor de ciencias
- Reposiciona tracking beacon a sat de comms
- Repara el sistema de Gamma Armory
- Agrega Walldrobes Faltantes

Meta:

- Repara accesos a oficina de payaso y mimo
- Evita bolts en puertas de mineria al iniciar el turno
- Repara lista de camaras visibles desde oficina de HoS y Warden
- Repara sistema de Gamma Armory
- Agrega Walldrobes Faltantes

## Changelog
:cl:
add: Reparaciones Delta
add: Reparaciones Meta
/:cl:

Spircen committed:
Reparaciones PT1

SagaStars reviewed (approved):
AUTOLATHE EN CIENCIAS VAMOOOOOS

Spircen committed:
Reparaciones Delta

Delta:
-Repara el sistema de Gamma Armory
Meta:
-Repara el sistema de Gamma Armory
-Repara el sistema de camaras de HoS y Warden

Spircen committed:
Walldrobes

DanaDririon reviewed (approved):
Cuando estés listo apachurrale no más

Spircen committed:
Scrubbers and RCS
